include/cosim/model_description.hpp

```diff
@@ -59,8 +59,8 @@ enum variable_variability
     continuous
 };
 
-/// Subset of Co-Simulation v2 capabilities in the XML schema
-struct cosim_capabilities
+/// A list of simulator capabilities
+struct simulator_capabilities
 {
     bool can_get_and_set_fmu_state{};
```

Comment:
Sorry, I missed this in my previous review, but these should not have `fmu` in their names, for the same reasons as mentioned earlier. Maybe just `can_save_state` and `can_export_state`, i.e., using the same terminology as the corresponding API functions?

Reply:
Btw, this is a stylistic preference, but I think it is clearer to be explicit about the initialisation of primitive variables like these, and use `= false` rather than `{}`.

Reply:
Updated as suggested. 

I will merge this PR after merging the proxyfmu PR.